### PR TITLE
feat: add contact and about us

### PR DESCRIPTION
**Pull request by kacper-adamczyk**

### CODE REVIEW

File: src/components/contact/Index.tsx [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+import Footer from "../common/Footer/Index";
+
+interface Props {
+  name: string;
+  about: string;
+}
+const Person = ({ name, about }: Props) => {
+  return (
+    <div className="persondiv">
+      <img className="chuj" src="footer.png" alt="" />
```
Comment (szymonrybczak) [inline src/components/contact/Index.tsx:14]: ?

File: src/components/contact/Index.tsx [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+      <h3 className="name">{name}</h3>
+      <p className="about">{about}</p>
+    </div>
+  );
+};
+
+const Contact = () => {
+  return (
+    <StyledContact>
+      <Header></Header>
```
Comment (szymonrybczak) [inline src/components/contact/Index.tsx:24]:
```suggestion
      <Header />
```

File: src/components/contact/Index.tsx [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+      <SubpageTitle title="Kontakt & O nas"></SubpageTitle>
+      <div className="about">czesc</div>
+      <div className="bigfuckingrecktangle">
```
Comment (szymonrybczak) [inline src/components/contact/Index.tsx:27]: ????????

File: src/components/contact/Index.tsx (hunk 1 above)
Comment (moskalakamil) [inline src/components/contact/Index.tsx:14]: ?

File: src/components/contact/Index.tsx [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+            name="Filip Wnęk"
+            about="Brunetki i blondynki, wszystkie was jebać dziwki
+          Żyjecie dla pieniędzy
+          Więc umrzecie kurwa w nędzy"
+          ></Person>
+        </div>
+        <h1>Kontakt</h1>
+        <h1 className="h11">Adres Email</h1>
+        <input type="text" className="mail" />
+        <h1 className="h11">Co ciekawego chcesz napisac?</h1>
```
Comment (szymonrybczak) [inline src/components/contact/Index.tsx:72]: i18n???????

File: src/components/contact/Index.tsx [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+        <textarea></textarea>
+      </div>
+      <Footer></Footer>
```
Comment (szymonrybczak) [inline src/components/contact/Index.tsx:75]:
```suggestion
      <Footer />
```

File: src/components/contact/Index.tsx [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+    margin: 5px auto;
+    display: grid;
+    position: relative;
+  }
+  .chuj {
+    height: 100px;
+    width: 100px;
+    border-radius: 60px;
+  }
+  .bigfuckingrecktangle {
```
Comment (moskalakamil) [inline src/components/contact/Index.tsx:110]: :O

File: src/pages/contact.tsx [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+import { serverSideTranslations } from "next-i18next/serverSideTranslations";
+
+const ContactPage = () => {
+  return (
+    <>
+      {" "}
+      <Contact />{" "}
+    </>
+  );
+};
```
Comment (szymonrybczak) [inline src/pages/contact.tsx:12]:
```suggestion
  return <Contact />
```

Comment (moskalakamil) [inline src/pages/contact.tsx:12]:
```suggestion
  return  <Contact />;
};
```

File: src/components/contact/Index.tsx [hunk 8]
```diff
@@ -0,0 +1,161 @@
...
+          adjsadbashjdbhsja dbshaudg usa ds vdsuagdfv gas vdsa vduyasg dhasdi
+          hasig duasg diuasgd yisa digg dhasdi hasig duasg diuasgd yisa digg
+          dhasdi hasig duasg diuasgd yisa dig
+        </p>
+        <div className="person">
+          <Person
+            name="Kacper Adamczyk"
+            about="Brunetki i blondynki, wszystkie was jebać dziwki
+          Żyjecie dla pieniędzy
+          Więc umrzecie kurwa w nędzy"
```
Comment (moskalakamil) [inline src/components/contact/Index.tsx:42]: :O

File: src/components/contact/Index.tsx [hunk 9]
```diff
@@ -0,0 +1,166 @@
...
+      <h3 className="name">{name}</h3>
+      <p className="about">{about}</p>
+    </div>
+  );
+};
+
+const Contact = () => {
+  return (
+    <StyledContact>
+      <Header></Header>
```
Comment (moskalakamil) [inline src/components/contact/Index.tsx:24]:
```suggestion
      <Header />
```

File: src/pages/contact.tsx (hunk 7 above)
Comment (moskalakamil) [inline src/pages/contact.tsx:12]:
```suggestion
  return (
    <>
      <Contact />
    </>
  );
};
```

**Review by TheTuso — APPROVED** [review]

**Review by szymonrybczak — APPROVED** [review]

**Review by fnwk — APPROVED** [review]

**Review by moskalakamil — APPROVED** [review]

**Pull request merged**